### PR TITLE
CORE-2307 shorten algorand truncation check

### PR DESCRIPTION
**Pull request by erer1243**
todo: example proving it works

### CODE REVIEW

File: hs/src/Reach/Connector/ALGO.hs [hunk 1]
```diff
@@ -1429,12 +1429,12 @@ cprim = \case
           -- [ v ]
           let ext i = cint (8 * i) >> op "extract_uint64"
           unless (trunc || pv == PV_Veri) $ do
-            dupn 3
-            -- [ v, v, v, v ]
-            let go i = ext i >> cint 0 >> asserteq
-            go 2
-            go 1
-            go 0
+            op "// Truncation check"
```
Comment (jeapostrophe) [inline hs/src/Reach/Connector/ALGO.hs:1432]: There's a function called `comment` for this

**Review by jeapostrophe — CHANGES_REQUESTED** [review]
Re: an example --- I think this already does it: https://github.com/reach-sh/reach-lang/blob/master/examples/uint256/index.mjs#L73

**Comment by erer1243** [comment]
you're right, the uint256 example does it. so, no new example

**Review by jeapostrophe — APPROVED** [review]

**Comment by erer1243** [comment]
this fails some of the closed code. do you want me to take a stab at that or just leave it to someone else?

**Comment by jeapostrophe** [comment]
Fix it, I messaged how in Slack

**Pull request merged**